src/web/mod.rs

```diff
@@ -103,7 +103,11 @@ impl CratesfyiHandler {
         router.get("/robots.txt", sitemap::robots_txt_handler, "robots_txt");
         router.get("/sitemap.xml", sitemap::sitemap_handler, "sitemap_xml");
         router.get("/opensearch.xml", opensearch_xml_handler, "opensearch_xml");
-        router.get("/std", rustdoc::std_redirector_handler, "std");
+        router.get("/alloc", rustdoc::RustLangRedirector::new("alloc"), "alloc");
```

Comment:
Maybe add an empty (and a comment) before and after this block so it's easier to spot it when reading?

Reply:
Sounds good.